syz-verifier/main.go

```diff
@@ -113,7 +113,7 @@ type runnerProgs map[int]*progInfo
 
 func main() {
 	var cfgs tool.CfgsFlag
-	flag.Var(&cfgs, "configs", "list of kernel-specific comma-sepatated configuration files ")
+	flag.Var(&cfgs, "configs", "list of kernel-specific comma-sepatated configuration files")
```

Comment:
To make this perfect we could say that this is a required flag and that the minimum number of configs is 2. Otherwise it prints help when executed w/o args, but it's unclear what's wrong or how to fix it.